packages/website/bin/generate-pages.js

```diff
@@ -16,6 +16,7 @@ module.exports = async ({
   docsPath,
   useManifests,
   webpackConfiguration,
+  showExamples,
```

Comment:
This needs to be called `showSubExamples` - the naming currently implies that it would try and render no examples at all.